Sources/MapboxNavigation/RouteLineType.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+enum RouteLineType {
+    
+    case source(isMainRoute: Bool = false, isSourceCasing: Bool = false)
+    
+    case route(isMainRoute: Bool = false)
```

Comment:
Do these associated values all need default values? Maybe it’s better if client code has to specify whether it’s main route explicitly, to avoid a bug where none of the routes is the main route.

Reply:
Agree, default parameters look more error prone.